incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -13,6 +13,7 @@ This document describes common tasks you might want to accomplish using HNC.
   * [Select namespaces based on their hierarchies](#use-select)
   * [Delete a subnamespace](#use-subns-delete)
   * [Organize full namespaces into a hierarchy](#use-full)
+  * [Move a subnamespace](#move-subnamespace)
```

Comment:
Let's move this to the "admin" section since this requires cluster-level privileges (editing a namespace). Also, can we change the name to "Convert subnamespace to full namespaces" and the anchor to `#admin-convert-subns` (all anchors either begin with `use-` or `admin-`).

Reply:
+1 to move this to `admin` section.

Since `move a subnamespace` contains two parts - `Convert subnamespace to full namespaces` and `Convert full namespaces to subnamespaces`, maybe consider having two separate ones instead of having 1?
(+1 to https://github.com/kubernetes-sigs/multi-tenancy/pull/1046/commits/1ebdd8b44f9172e28caa6bd28aaff77c184ede9d#r476477948)